spec/models/manageiq/providers/azure/cloud_manager_spec.rb

```diff
@@ -1,3 +1,4 @@
+require 'azure-armrest'
 describe ManageIQ::Providers::Azure::CloudManager do
```

Comment:
No worries, @bdunne showed me the test is actually using the constant `Azure::Armrest::UnauthorizedException` on line 57.  👍